### ISSUE TITLE
Redeploy nodes using our single Compose recipe

### ISSUE THREAD

**Issue by alyssadai** [neurobagel/planning#147]
- [x] `st-viateur`
  - [x] QPN
  - [x] PPMI (recipe switched over, but not launched)
- [x] `homemade`
  - [x] QPN & PPMI
  - [x] f-API
- [x] `fairmount`
  - [x] OpenNeuro
  - [x] INDI
  - [x] f-API

**Comment by surchs** [comment]
As you redeploy, could you also check to make sure we're not including OpenNeuro test any more:



![Image](https://github.com/neurobagel/planning/assets/1302022/4bc8476a-d782-445c-8647-bcc2d2835f09)

**Comment by surchs** [comment]
Thanks a lot @alyssadai for doing the redeployment!

Some notes on the machines (feel free to spin off into separate issues):

- viateur
  - [ ] `openneuro_recipe` services are still running - I assume that's a test instance? I'd say either shut down or rename to test
  - [ ] `/home/ubuntu/stardog-home` should either be removed or renamed (ideally removed)
  - [ ] I really like `/home/ubuntu/graphdb-homes` as a central place for all of the graphs (assuming that's how it is intended). However `/home/ubuntu/graphdb_root_openneuro` sticks out there. So let's remove it and if we do still need it -> incorporate into graphdb-homes
- fairmount
  - [ ] `graphdb_root_qpn` like above: let's remove it and if we do still need it -> incorporate into graphdb-homes
  - [ ] I really like how clean `data` is on viateur. It seems like we can wipe the stuff in `data` that isn't a project specific upload dir. Anything truly irreplaceable can go in hackmd or something like that
  - [ ] `stardog-home` `stardog_root` should both go
  - `projects` dirs that should be removed
    - [ ] `old_stardog_licenses`
    - [ ] `scratch`
    - [ ] `nipoppy-qpn` -> or move it to some other dir if it is not meant to spin up a node. The `projects` folder should be a place you quickly see all the docker stacks we have ready to go on the machine  I think
  - [ ] can we try to name the `projects/<dir-name>` consistently? Some are called node: `indi_node` others recipe `qpn_recipe` and others have no suffix `federation`
  - [ ] I don't understand what `projects/federation` does. It only seems to have a local nodes list in it
  - [ ] `projects/qpn_recipe` seems to not use a clone of the new single recipe repo. Is that current? The main goal here is to make sure that if the machine were to go down and we need to restart the nodes, we know immediately what dir to go into to do so - and each node generally starts up the same way

This is starting to look very good. I think once we close this issue, we can probably even have the nodes auto-restart and then make this more or less hands-off (tbd).

**Comment by alyssadai** [comment]
Thanks for the feedback @surchs!

I've made your suggested changes, and also added a top level `scratch/` directory on each machine so we still have a location to test tools/deployments w/o cluttering `projects/`.

The data and code for `digest` have been moved into a top-level `tools/` directory on `fairmount`. 

> I don't understand what projects/federation does. It only seems to have a local nodes list in it
Renamed to `projects/neurobagel_public_federation/` and switched to using our `local_federation` recipe inside it.

If all looks good, feel free to close!
